site/pages/create-disruption.tsx

```diff
@@ -1,24 +1,104 @@
 import React, { ReactElement } from "react";
 import DateSelector from "components/DateSelector";
 import TimeSelector from "components/TimeSelector";
-import { BaseLayout } from "components/layout/Layout"
+import { BaseLayout } from "components/layout/Layout";
 import { DisruptionValidity } from "interfaces";
 
-const title = 'Create Disruptions';
-const description = 'Create Disruptions page for the Create Transport Disruptions Service';
+const title = "Create Disruptions";
+const description = "Create Disruptions page for the Create Transport Disruptions Service";
 
 interface CreateDisruptionProps {
-    inputs: DisruptionValidity
+    inputs: DisruptionValidity;
 }
 
+enum Reason {
+    roadWorks = "Road Works",
+    vandalism = "Vandalism",
+    routeDiversion = "Route Diversion",
+    specialEvent = "Special Event",
+}
+
+const getReasonOptions = (): JSX.Element[] => {
+    let options = [];
+
+    for (let element in Reason) {
+        options.push(<option value={element}>{Reason[element as keyof typeof Reason]}</option>);
+    }
+
+    return options;
+};
+
 const CreateDisruption = ({ inputs }: CreateDisruptionProps): ReactElement => {
     return (
         <BaseLayout title={title} description={description}>
             <>
                 <div className="govuk-form-group">
-                    <h1 className="govuk-heading-l">
-                        When is the disruption?
-                    </h1>
+                    <h1 className="govuk-heading-xl">Create a new Distruption</h1>
+                    <fieldset className="govuk-fieldset">
```

Comment:
I assume we should have a fieldset for each sub section

Reply:
Any related inputs should be in a fieldset, so for this section only the planned and unplanned fields really need to be in one

Reply:
done